src/types/payout.ts

```diff
@@ -0,0 +1,22 @@
+// available tokens for payouts
+export const PAYMENT_TOKEN_PER_NETWORK: Record<string, { rpc: string; token: string; symbol: string }> = {
+  "1": {
+    rpc: "https://rpc.gnosischain.com",
+    token: "0x6B175474E89094C44Da98b954EedeAC495271d0F",
+    symbol: "DAI",
+  },
```

Comment:
This doesn't seem right. Network ID 1 is mainnet. The RPC clearly is for gnosis chain. Perhaps try and find a nice mainnet RPC?

Reply:
Good point, also symbol should not be DAI but ETH isn't it?
`pay.ubq.fi` seems to be using https://rpc.mevblocker.io for its RPC which seems to have low latency, a good privacy and score according to Chainlist as well.

Reply:
The token address indeed represents DAI on mainnet.